src/api/status/public/buildLog/api.js

```diff
@@ -0,0 +1,37 @@
+export async function checkBuildStatus() {
+  try {
+    const res = await fetch('/v1/status/status');
```

Comment:
This is going to need to point to the autodeployment server.

We're probably going to need to do some URL trickery to figure out which server (local vs. staging vs. production) to use, where you take the current URL and add `:4000/status` to the end.

Reply:
So I guess the only way to test it is pushing to dev?